core/config/project_settings.cpp

```diff
@@ -1588,6 +1588,8 @@ ProjectSettings::ProjectSettings() {
 
 	GLOBAL_DEF_BASIC("gui/common/snap_controls_to_pixels", true);
 	GLOBAL_DEF_BASIC("gui/fonts/dynamic_fonts/use_oversampling", true);
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "gui/fonts/min_atlas_texture_size", PROPERTY_HINT_RANGE, "8,14,1"), 8);
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "gui/fonts/max_atlas_texture_size", PROPERTY_HINT_RANGE, "8,14,1"), 11);
```

Comment:
Somewhat unorthodox for an user interface, especially given these are not the exact value being used. Should probably be an enum of valid values?

Reply:
It's only 7 values so enum make sense, changed.